app/views/catalog/index.html.erb

```diff
@@ -10,38 +10,10 @@
 
 <div id="content">
 
-    <% unless has_search_parameters? %>
-	    <%# if there are no input/search related params, display the "home" partial -%>
-	    <%= render 'home' %>
-    <% else %>
-
-      <h2 class="sr-only top-content-title"><%= t('blacklight.search.search_results_header') %></h2>
-
-      <% @page_title = t('blacklight.search.title', :application_name => application_name) %>
-
-
-      <% provide(:head) do -%>
-        <%= render_opensearch_response_metadata %>
-        <%= auto_discovery_link_tag(:rss, url_for(params.merge(:format => 'rss')), :title => t('blacklight.search.rss_feed') ) %>
-        <%= auto_discovery_link_tag(:atom, url_for(params.merge(:format => 'atom')), :title => t('blacklight.search.atom_feed') ) %>
-      <% end -%>
-
-
-      <%= render 'search_header' %>
-
-      <h2 class="sr-only"><%= t('blacklight.search.search_results') %></h2>
-
-      <%- if @response.empty? %>
-        <%= render "zero_results" %>
-      <%- elsif render_grouped_response? %>
-        <%= render_grouped_document_index %>
-      <%- else %>
-        <%= render_document_index %>
-      <%- end %>
-
-	    <%= render 'results_pagination' %>
-
-
-    <% end %>
+  <% if has_search_parameters? || @document_list.count > 0 %>
```

Comment:
why are we testing for `@document_list.count`? Why aren't we just doing the blacklight default behavior? https://github.com/projectblacklight/blacklight/blob/master/app/views/catalog/index.html.erb#L6